WebDriverAgentLib/Utilities/FBAppiumActionsSynthesizer.m

```diff
@@ -195,9 +197,12 @@ - (nullable instancetype)initWithActionItem:(NSDictionary<NSString *, id> *)item
   self = [super initWithActionItem:item application:application atPosition:atPosition offset:offset error:error];
   if (self) {
     _pressure = nil;
+    _pressDuration = nil;
```

Comment:
there should be no duration in press. The duration is added automatically if pause/wait is called after press or longPress is used

Reply:
Probably, it should be done in another way specifically for the force touch. With different experiments I figured out that duration and pressure override of the event do the job.